cmake/Modules_CUDA_fix/upstream/FindCUDA.cmake

```diff
@@ -124,7 +124,7 @@
 #      CUDA_ADD_LIBRARY, CUDA_ADD_EXECUTABLE, or CUDA_WRAP_SRCS.  Flags used for
 #      shared library compilation are not affected by this flag.
 #
-#   CUDA_PROPAGATE_HOST_FLAGS_BLACKLIST (Default "")
+#   CUDA_PROPAGATE_HOST_FLAGS_DENYLIST (Default "")
```

Comment:
aren't FindCUDA files just come from upstream somewhere? Though it seems like an obscure flag

Reply:
Reverted.